core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java

```diff
@@ -93,6 +96,46 @@
             ApplicationProtocolNames.HTTP_2,
             ApplicationProtocolNames.HTTP_1_1);
 
+    /*
+     * Validate {@code sslContext} is configured properly. If {@code sslContext} is configured as client
+     * context, or key store password is not given to key store when {@code sslContext} is created using key
+     * manager factory, the validation will fail and an exception will be raised.
```

Comment:
nit: an exception -> an `{@link SSLException}`

Reply:
I fixed it. Thank you for review :+1:

Reply:
It has to be `an {@link SSLException}`. You omitted `an` :wink:

Reply:
Oops, I should've double checked it.
Please let me fix it.